README.md

```diff
@@ -60,6 +60,7 @@ type XhrOptions = String | {
     withCredentials: Boolean?,
     responseType: String?,
     beforeSend: Function?
+    qs: Any?
```

Comment:
I'll leave it as-is for now. Y'all can hash this out in the v3 branch post-merge?